src/main/kotlin/org/jetbrains/bio/genome/Annotations.kt

```diff
@@ -103,6 +119,67 @@ object Repeats {
 
         return builder.build()
     }
+
+    private fun parseHs1(
```

Comment:
It would be better to add this code to the `AnnotationsHs1.kt` file and delegate it to specialized functions from this file. It will make adding the next extraordinary genome easier.

Reply:
Done.